cardano-api/src/Cardano/Api/Query.hs

```diff
@@ -381,7 +381,6 @@ newtype CurrentEpochState era = CurrentEpochState (Shelley.EpochState (ShelleyLe
 
 decodeCurrentEpochState
   :: forall era. Ledger.Era (ShelleyLedgerEra era)
-  => Share (Core.TxOut (ShelleyLedgerEra era)) ~ Interns (Shelley.Credential 'Shelley.Staking (Ledger.Crypto (ShelleyLedgerEra era)))
```

Comment:
Interesting, what prompted this change?

Reply:
`ghc-9.2.4` flagged that as an unused constrained and given that it compiles with `ghc-8.10.7` flagging it as unused is correct.